app/views/school/la_funded_places/laptop_types.html.erb

```diff
@@ -0,0 +1,40 @@
+<%- title = t('page_titles.iss.laptop_types') %>
+<%- content_for :title, title %>
+<%- content_for :before_content do %>
+  <%- school_breadcrumbs items: [ { t('page_titles.iss.order_devices') => order_devices_school_path(@school)}, title], school: @school, user: impersonated_or_current_user %>
+<%- end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= title %>
+    </h1>
+    <p class="govuk-body">
+      We recommend checking with schools, families or pupils as soon as possible to find out which device type and IT settings would be most suitable.
```

Comment:
... checking with your IT team, schools, families ...

Reply:
Done